CMakeLists.txt

```diff
@@ -23,8 +23,8 @@ if (NOT DEFINED USE_LOG4CPP)
   option(USE_LOG4CPP "Enable LOG4CPP" ON)
 endif()
 
-option(RUN_APOLLO_TESTS "Enable Apollo tests run" ON)
-option(KEEP_APOLLO_LOGS "Retains logs from replicas in separate folder for each test in build/tests/apollo/logs" ON)
+option(RUN_APOLLO_TESTS "Enable Apollo tests run" OFF)
```

Comment:
why have you disabled these options

Reply:
The Apollo tests in native builds always fails in my testing. This is because during the build they look for certain files in locations that would be available in the docker builds. There are instructions in the README file on how to run the Apollo tests. So instead of running failing tests during native build (via Cmake) which takes a lot of time, users who want to run native build can run Apollo tests via instructions in the README file and save time on running the tests via make test.

Reply:
AFAIR there are two scripts addressed by an absolute path docker path `/conncord/*`: one for transaction signing and one for certificates generation. This should be fixed.

Reply:
In that case I will enable this flag for now. I will raise the issue of the failing Apollo tests as a separate issue.